.github/CODEOWNERS

```diff
@@ -1,7 +1,8 @@
 * @jzuniga-amd @tfalders @cgmb @qjojo @EdDAzevedo @jmachado-amd
 # Documentation files
-docs/* @ROCm/rocm-documentation @jzuniga-amd @tfalders @cgmb @qjojo @EdDAzevedo @jmachado-amd
-*.md @ROCm/rocm-documentation @jzuniga-amd @tfalders @cgmb @qjojo @EdDAzevedo @jmachado-amd
-*.rst @ROCm/rocm-documentation @jzuniga-amd @tfalders @cgmb @qjojo @EdDAzevedo @jmachado-amd
+docs/ @ROCm/rocm-documentation
+*.md @ROCm/rocm-documentation
+*.rst @ROCm/rocm-documentation
+.readthedocs.yaml @ROCm/rocm-documentation
```

Comment:
Shouldn't the rocSOLVER developers also be listed as code owners on these files?

Reply:
See, e.g., https://github.com/ROCm/rocSOLVER/pull/668

Reply:
Would you like to keep developers as code owners of documentation files?

My original intention was to have PRs solely for documentation flag just the documentation team. My understanding is that developers will be added as reviewers if the PR changes anything not related to the documentation files listed in this CODEOWNERS file.

eg: kept in line 8 since header file changes are usually code changes, but also may include doxygen documentation changes (library/include/ @ROCm/rocm-documentation @jzuniga-amd @tfalders @cgmb @qjojo @EdDAzevedo @jmachado-amd)

Reply:
> Would you like to keep developers as code owners of documentation files?

Yes.

Reply:
[Add devs as code owners for all files](https://github.com/ROCm/hipSOLVER/pull/316/commits/df69a04afdd92822a7c702b6a490287f7fdf43fd)